File: Code Chef/JavaScript/500-599/585 Nearest Exit NEARESTEXIT.js
let input = '';
process.stdin.on('data', (data) => input += data);

const strToInt = (strArray) => {
    strArray.shift();
    const intArray = strArray.map((strItem) => Number(strItem.trim()));
    
    return intArray;
};

const whichExitToUse = (seatNo) => {
    (seatNo < 51)
        ? console.log('LEFT')
        : console.log('RIGHT');
}

process.stdin.on('end', () => {
    const testcases = strToInt(input.trim().split('\n'));
    
    testcases.forEach((seatNo) => {
        whichExitToUse(seatNo);
    });
});

File: Code Chef/C Programming/500-599/585 Nearest Exit NEARESTEXIT.c
#include <stdio.h>
int main() 
{
    int t,i,seat;
    scanf("%d",&t);
    for(i=0;i<t;i++)
    {
        scanf("%d",&seat);
        if(seat<=50)
        {
            printf("LEFT\n");
        }
        else
        {
            printf("RIGHT\n");
        }
    }
	return 0;
}